### PR TITLE
[SCHEDULED for Dec 7, do not merge before] feat(blog): greenkeeper

### PR DESCRIPTION
**Pull request by janl**
Part 3 in our little series.

### CODE REVIEW

File: _posts/2015-12-7-greenkeeper.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Greenkeeper"
+author: Jan
+layout: blog-post-tgif-ish
+comments: true
+---
+
+This is Part 3 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
```
Comment (varjmes) [inline _posts/2015-12-7-greenkeeper.md]: "of of" -> "of"

File: _posts/2015-12-7-greenkeeper.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Stop worrying about npm dependencies breaking your build: Greenkeeper saves you time and money by automating dependency updates and notifying you of failed builds so you can spend your development time actually making things.
```
Comment (varjmes) [inline _posts/2015-12-7-greenkeeper.md]: Might be nice to have a picture of the greenkeeper logo somewhere around this paragraph? :)

File: _posts/2015-12-7-greenkeeper.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+3. The third release cleaned up a lot of little things and made the backend a lot more reliable. Outward facing, we now support your choice of tabs vs. spaces in your package.json (one of you is wrong, but we don’t care ;) and we added support for enabling Greenkeeper on repo forks, to support more workflows more flexibly.
+
+4. This time, we didn’t change anything, but we made all our plans more valuable: every plan on Greenkeeper now comes with support for one private repo. This allows anyone to try out Greenkeeper in any setup without much hassle.
+
+As a result of running this for about 10 weeks now, we are learning a lot about the npm ecosystem. We’ve already sent more than TBD_AT_TIME_OF_PUBLISH pull requests, saving an estimated TBD_AT_TIME_OF_PUBLISH months of developer time,
+
+// screen shot github streak TBD_AT_TIME_OF_PUBLISH
```
Comment (varjmes) [inline _posts/2015-12-7-greenkeeper.md]: Dont forget to update this!

**Comment by janl** [comment]
Thanks all for the review! — I’ll merge this now, PR-efforts starts tomorrow.

**Pull request merged**